pages/account/statistics.js

```diff
@@ -16,6 +16,7 @@ import Alert from "../../components/Alert";
 import Page from "../../components/Page";
 import PageHead from "../../components/PageHead";
 import { abbreviateNumber } from "../../services/utils/abbreviateNumbers";
+import BasicCards from "../../components/statistics/BasicCards";
 
 export async function getServerSideProps(context) {
   const session = await unstable_getServerSession(
```

Comment:
Just as a reminder, on newer versions of next-auth this function is not unstable, so we might want to update ✅

Reply:
good spot, yes this has been deprecated 👍 we will need a separate PR to update all of these